src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -397,272 +346,259 @@
+                                                                     Style="{StaticResource VariableTextBoxStyle}"
+                                                                     GotFocus="TextBoxGotFocus"
+                                                                     KeyDown="TextBoxKeyDown"
+                                                                     LosingFocus="TextBoxLosingFocus"
+                                                                     Text="{x:Bind Step, Mode=OneWay}"/>
+                                                        </StackPanel>
+                                                        <StackPanel Grid.Column="2" Orientation="Horizontal">
+                                                            <TextBlock x:Uid="MaxTextBlock"
+                                                                       Padding="0,0,0,8"
+                                                                       VerticalAlignment="Bottom"
+                                                                       FontWeight="SemiBold"/>
+                                                            <TextBox x:Name="MaxTextBox"
+                                                                     MaxWidth="96"
+                                                                     Style="{StaticResource VariableTextBoxStyle}"
+                                                                     GotFocus="TextBoxGotFocus"
+                                                                     KeyDown="TextBoxKeyDown"
+                                                                     LosingFocus="TextBoxLosingFocus"
+                                                                     Text="{x:Bind Max, Mode=OneWay}"/>
+                                                        </StackPanel>
+                                                    </Grid>
+                                                </StackPanel>
+                                            </DataTemplate>
+                                        </ListView.ItemTemplate>
+                                    </ListView>
+                                </Flyout>
+                            </Button.Flyout>
+                        </Button>
+
+                        <ToggleButton x:Name="ActiveTracing"
+                                      MinWidth="40"
+                                      Margin="0,-1"
+                                      Style="{ThemeResource ThemedGraphToggleButtonStyle}"
+                                      BorderThickness="0,0,1,0"
+                                      AutomationProperties.Name="{x:Bind local:GraphingCalculator.GetTracingLegend(GraphingControl.ActiveTracing), Mode=OneWay}"
+                                      Checked="ActiveTracing_Checked"
+                                      IsChecked="{x:Bind GraphingControl.ActiveTracing, Mode=TwoWay}"
```

Comment:
This still hits the same trap we talked about. To repro try this:

1. Toggle Active Trace on
2. Press shift +tab until the GraphControl has focus, verify that you can move the red dot with the arrow keys
3. Click Active Trace off
4. Notice that it instantly turns on, then back on again

When the focus bug is fixed, steps 2 and 4 will give focus to the Graph Control, causing the loop to continue.

Reply:
good catch! Fixed!